src/commands/delete.js

```diff
@@ -0,0 +1,48 @@
+const utils = require('../utils');
+
+const _delete = (context, version) => {
+  if (!version) {
+    context.line('Error: No version - provide a version like "1.0.0"...\n');
+    return Promise.resolve(true);
+  }
+  return utils.checkCredentials()
+    .then(() => utils.getLinkedApp())
+    .then((app) => {
+      context.line(`Preparing to delete version ${version} of your app "${app.title}".\n`);
```

Comment:
I'm not sure of the extra `\n` I see the `context.line` is just `console.log`. That should already add a new line?